src/app/components/forms/chat/mods/FormChatMods.vue

```diff
@@ -0,0 +1,322 @@
+<script lang="ts">
+import { computed, onMounted, onUnmounted, PropType, ref, toRefs } from 'vue';
+import { arrayRemove } from '../../../../../utils/array';
+import { Api } from '../../../../../_common/api/api.service';
+import { showErrorGrowl, showSuccessGrowl } from '../../../../../_common/growls/growls.service';
+import AppIllustration from '../../../../../_common/illustration/AppIllustration.vue';
+import AppLoading from '../../../../../_common/loading/AppLoading.vue';
+import AppLoadingFade from '../../../../../_common/loading/AppLoadingFade.vue';
+import { ModalConfirm } from '../../../../../_common/modal/confirm/confirm-service';
+import { Screen } from '../../../../../_common/screen/screen-service';
+import AppScrollInview, {
+	ScrollInviewConfig,
+} from '../../../../../_common/scroll/inview/AppScrollInview.vue';
+import { useCommonStore } from '../../../../../_common/store/common-store';
+import { $gettext, $gettextInterpolate } from '../../../../../_common/translate/translate.service';
+import { User } from '../../../../../_common/user/user.model';
+import { illNoCommentsSmall } from '../../../../img/ill/illustrations';
+import { tryGetRoomRole } from '../../../chat/client';
+import { ChatRole } from '../../../chat/role';
+import { ChatRoom } from '../../../chat/room';
+import { ChatUser } from '../../../chat/user';
+import AppChatList from '../../../chat/_list/AppChatList.vue';
+import { useGridStore } from '../../../grid/grid-store';
+import AppFiresideChatModsItem from './AppFiresideChatModsItem.vue';
+
+export type ChatModSection = 'currentMods' | 'friends' | 'chat';
+</script>
+
+<script lang="ts" setup>
```

Comment:
this component needs to make sure it gets a watch on the member collection for the room, otherwise it might not have the up to date members